docs/resource-schemas.md

```diff
@@ -22,6 +22,16 @@ library:
       name: Storybook
       action: link
       url: https://react.carbondesignsystem.com
+  navData:
+    - title: A page
+      path: '/a-page'
```

Comment:
Should all of these paths include file extension? E.g. `path: '/a-page.mdx'`.

Reply:
The path where to find the file in the remote repo. We could also use that for the web app path, or we could slugify title for that.